netlify.toml

```diff
@@ -55,6 +55,7 @@ redirects = [
     {from = "/models/comparison", to = "/models", force = true},
     {from = "/api/#section-cython", to = "/api/cython", force = true},
     {from = "/api/#cython", to = "/api/cython", force = true},
+    {from = "/api/architectures#TextCatCNN", to = "/api/legacy#TextCatCNN_v2", force = true},
```

Comment:
Do we want to remove this now since it's kind of misleading?

Reply:
Done.